package.json

```diff
@@ -52,7 +52,7 @@
     "@types/ssh2": "^0.5.36",
     "amazon-cognito-identity-js": "^3.0.2",
     "async": "^2.6.1",
-    "aws-api-gateway-client": "https://github.com/aregier/aws-api-gateway-client/releases/download/v0.2.16-beta/aws-api-gateway-client-0.2.16.tgz",
```

Comment:
I'm not sure what I did incorrectly with the tarball, but it wasn't working at runtime.
I believe the steps I followed were
```
git clone https://github.com/kndt84/aws-api-gateway-client.git
npm install
npm run prepublish
npm pack
```

Reply:
The Readme doesn't give too much clue about the official dev steps, and the travis CI file looks fairly simple:
https://github.com/kndt84/aws-api-gateway-client/blob/master/README.md